include/translated-object.php

```diff
@@ -518,8 +526,8 @@ public function set_translation_in_mass( $translations ) {
 		$description = array();
 		$count       = array();
 
-		foreach ( $translations as $t ) {
-			$term = uniqid( 'pll_' ); // The term name.
+		foreach ( $translations as $k => $t ) {
+			$term = is_string( $k ) ? "pll_{$k}" : uniqid( 'pll_' ); // The term name.
```

Comment:
Why prefixing it? Can't we assume the calling code have a proper name already?

Reply:
We can assume it has IMHO. I can remove it if you prefer.

Reply:
Done in [d41a193](https://github.com/polylang/polylang/pull/1664/commits/d41a1933a8d8f69dc847596d6ad4e0ac549118b9).